website/docs/product/telemetry.md

```diff
@@ -8,58 +12,102 @@ Appsmith is a lightweight proxy and **does not capture** any data returned by yo
 
 ### Keep-alive ping
 
-The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used.
+The Appsmith server sends a keep-alive ping every 6 hours to indicate that it's still running without any errors & captures anonymous usage data around which plugins are being used. This data is collected irrespective of whether telemetry is turned on or off.
+
+```javascript title="sample keep-alive ping"
+{
+    "event": "FETCH_RELEASE_NOTES",
+    "properties": {
+        "time": 1704784217.602,
+        "distinct_id": "609ce8fb4092701c69df9846",
+        "$identified_id": "609ce8fb4092701c69df9846",
+        "$import": true,
+        "$insert_id": "75ddda7c-ba47-41e0-9578-7d3afaf05284",
+        "$lib_version": "3.3.1",
+        "$mp_api_endpoint": "api.mixpanel.com",
+        "$mp_api_timestamp_ms": 1704784233604,
+        "$source": "segment",
+        "$user_id": "609ce8fb4092701c69df9846",
+        "edition": "CE",
+        "id": "609ce8fb4092701c69df9846",
+        "instanceId": "609ce8fb4092701c69df9846",
+        "mp_lib": "Segment Actions: analytics-java",
+        "mp_processing_time_ms": 1704784233665,
+        "originService": "cloud-services",
+        "segment_source_name": "cloud-services",
+        "type": "fetch-release-notes",
+        "version": "v1.9.15"
+    }
+}
+```
 
-**Sample event**
+### Server setup ping
+Appsmith server sends a ping first time a new instance is created. This data is collected irrespective of telemetry is turned on or off.
```

Comment:
"This data is collected irrespective of **whether** telemetry is turned on or off (just trying to be consistent with the language above)

Reply:
good catch